tests/osgi-tests/src/test/java/org/wso2/is/portal/user/client/api/unit/test/ChallengeQuestionManagerClientServiceTest.java

```diff
@@ -120,34 +123,33 @@ public void testSetChallengeQuestionForUser() throws UserPortalUIException, User
 
         users.add(user);
 
-       /* try {
+        try {
             challengeQuestionManagerClientService.setChallengeQuestionForUser(users.get(0).getUserId(),
-                    challengeQuestions.get(0).getQuestionId(),
-                    challengeQuestions.get(0).getQuestionSetId(), "Answer1");
+                    challengeQuestions.get(0).getQuestionId(), new String(Base64.getEncoder().encode
+                            (challengeQuestions.get(0).getQuestionSetId().getBytes(Charset.forName("UTF-8"))),
+                            Charset.forName("UTF-8")), "Answer1");
```

Comment:
Fixed in https://github.com/wso2/product-is/pull/1131/commits/d0a614a94b03694414c4c2f2b476549a985dd771